code/game/mecha/mecha.dm

```diff
@@ -68,7 +68,7 @@
 	var/internal_damage = 0 //contains bitflags
 
 	var/list/operation_req_access = list()//required access level for mecha operation
-	var/list/internals_req_access = list(ACCESS_ENGINE,ACCESS_ROBOTICS)//REQUIRED ACCESS LEVEL TO OPEN CELL COMPARTMENT

```

Comment:
why no engineering access?

Reply:
wait i forgot what the actual content of the PR said. i'm annoyed by this because it's not very atomic but whatever